File: Assignment-2/code/part1-snippets/calculate-sum-fixed.py
# Python : Calculate the sum of an array
# Fix: the letter o on line 3 of the broken file is the digit zero.
def calculate_sum(arr):
    total = 0
    for num in arr:
        total += num
    return total


numbers = [1, 2, 3, 4, 5]
result = calculate_sum(numbers)
print("Sum in Python:", result)

File: Assignment-2/code/part1-snippets/calculate-sum-fixed.js
// JavaScript : Calculate the sum of an array
// Fixes: the letter o is the digit zero, and the call site is one name, calculateSum, not two.
function calculateSum(arr) {
  let total = 0;
  for (let num of arr) {
    total += num;
  }
  return total;
}

let numbers = [1, 2, 3, 4, 5];
let result = calculateSum(numbers);
console.log("Sum in JavaScript:", result);
